cumulusci/cumulusci.yml

```diff
@@ -410,7 +410,20 @@ tasks:
         description: Remove specified XML elements from one or more metadata files
         class_path: cumulusci.tasks.metadata.modify.RemoveElementsXPath
         group: Salesforce Metadata
-
+    disable_tdtm_trigger_handlers:
+        class_path: cumulusci.tasks.salesforce.trigger_handlers.SetTDTMHandlerStatus
+        description: "Disable specified TDTM trigger handlers"
+        group: Utilities
```

Comment:
Maybe `group: Salesforce.org`?

Reply:
What about `group: NPSP/EDA`? I don't want to give the implication that they're just for our use (extenders of the packages need this too).